renderer/pages/setup/4.tsx

```diff
@@ -62,6 +62,14 @@ const ThirdStep = () => {
         </Typography>
       </Grid>
 
+      {appNames.includes("Grid") && (
+        <Grid item xs={8}>
+          <Typography variant="body1" color="error">
+            This will upload phrases from every Grid user on this computer.
```

Comment:
Can you reword this slightly to make it clear that the upload doesn't happen immediately? Maybe just change "upload" -> "add".